src/Mod/PartDesign/Gui/Command.cpp

```diff
@@ -1738,6 +1738,17 @@ void prepareTransformed(Gui::Command* cmd, const std::string& which,
         // TODO Wjat that function supposed to do? (2015-08-05, Fat-Zer)
         func(FeatName, features);
 
+        PartDesign::Body *pcActiveBody = PartDesignGui::getBody(true);
+
+        if(pcActiveBody) {
+            App::DocumentObject* prevSolid = pcActiveBody->Tip.getValue();
+
+            if (prevSolid) {
+                // Set the tip of the body
+                Gui::Command::doCommand(Gui::Command::Doc,"App.activeDocument().%s.BaseFeature = App.activeDocument().%s",
```

Comment:
This needs investigation. Back when I wrote a tutorial for python partdesign feature, I found out that Body writes BaseFeature of any PD feature as soon as it is added. So in theory this code should not be needed.
https://forum.freecadweb.org/viewtopic.php?f=22&t=21097

Reply:
found it! It is because Body::isMemberOfMultiTransform thinks the newly created transform is a member of a multitransform, but it is not. So this fix looks like a reasonable workaround.